language-design.md

```diff
@@ -111,7 +233,12 @@ you do so? If not, why not?
 Briefly describe how you split up the work for this assignment.
 
 **Response**
+We read all the articles separately and came together to write the responses.
+Both of us talked about our ideas before writing each response. Sam typed up
+the responses, while Tiffany focused on the wording of responses and the big
+picture ideas we wanted to get across.
 
+Separately, we both also proofreaded and added to some responses on our own.
 
 
 ---
```

Comment:
I had agreements and disagreements with your responses that were interesting. Overall, great job!